### PR TITLE
Add Node Development Projects Extension

### PR DESCRIPTION
**Pull request by grantvanhorn**
## Description
Set a node development directory. Node Dev command then allows users to see a list of their node projects. There are two actions to take on each project listed.
1. Open in VSCode
2. Open project file path in Hyper Terminal

## Type of change
- New extension

## Screencast
https://youtu.be/tOBOsMrEPsI

## Checklist

- [ x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

File: extensions/node-development-projects/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "version": "1.0.0",
+  "title": "Node Development Projects",
+  "description": "Perform actions on Node projects in your development directory. Open projects in VSCode or Hyper Terminal.",
+  "icon": "node-title.png",
+  "author": "grantcodesgg",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "node-dev",
+      "title": "Node Dev",
```
Comment (mattisssa) [inline extensions/node-development-projects/package.json:13]:
```suggestion
      "title": "Search Node Project",
```

**Review by mattisssa — COMMENTED** [review]
Hey! Thanks for the contribution, I left a small comment about naming. Leaving the main review to @pernielsentikaer now

File: extensions/node-development-projects/src/node-dev.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+              icon="vscode.png"
+              target={target}
+              application="Visual Studio Code"
+            />
+            <OpenAction
+              title="Open Hyper at this projects path"
+              icon="hyper.png"
+              target={target}
+              application="Hyper"
+            />
```
Comment (pernielsentikaer) [inline extensions/node-development-projects/src/node-dev.tsx:150]: I don't have Visual Code or Hyper, so these two actions won't work for me – can you come up with a way to test if it's installed before showing them?

You could also use the below actions:

<ShowInFinderAction path={target} />
<OpenWithAction path={filePath} />

File: extensions/node-development-projects/package.json (hunk 1 above)
Comment (pernielsentikaer) [reply extensions/node-development-projects/package.json:13]: Plural, with s 😄

File: extensions/node-development-projects/package.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "node-development-projects",
+  "version": "1.0.0",
+  "title": "Node Development Projects",
+  "description": "Perform actions on Node projects in your development directory. Open projects in VSCode or Hyper Terminal.",
```
Comment (pernielsentikaer) [inline extensions/node-development-projects/package.json:6]: This should maybe be modified after we find a better solution for the actions

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @grantvanhorn 👋 

Awesome extension, I see a lot of potential in it 💪 We left some feedback for you to address

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by mattisssa** [comment]
Hello @grantvanhorn! What is the status of this PR? Can you please take a look at the comments above?

**Comment by fe9lix** [comment]
Guys, I'm closing this because of inactivity. Feel free to re-open later again!

**Pull request closed without merging**